### PR TITLE
feat: clean up create-package script

### PR DESCRIPTION
**Pull request by huhgawz**
## Jira

[[DS] Clean up create-package script](https://littlespoon.atlassian.net/browse/PCH-1252)

## Motivation

`dependabot` created #1018. However, it was impossible to bump `lerna` dependency due to [an error](https://github.com/little-spoon-dev/design-system/actions/runs/6011220168/job/16304161348). 

Consequently, after reading https://lerna.js.org/docs/legacy-package-management, @huhgawz applied some changes to #1018 in order to fix the errors that were making impossible to bump `lerna` dependency to the `7.x` version:

1. `useWorkspaces` option was removed from the `lerna.json` file (fixed in 661f3e69e2001b7c0d131a2faa7b122d98392118)

2. `add` command was removed from Lerna CLI (fixed in c87837222d2252c23d0bd803e8cd87b1c2e4ccb2, 560b31776b71fc8aeb070d493ae45b31a64d3214, 337c543af5e52c9ac1b3731d368b3a4ee7efd6a5 and 7fab49b17ceddf6240fb8cf5d80861b4f7d27b73)

Unfortunately, when all tests successfully passed in #1018, `mergify` automatically merged #1018 which included some commented out code. This PR is about removing that commented out code and is also an opportunity to code review [changes in #1018](https://github.com/little-spoon-dev/design-system/pull/1018/files).

## Test Plan

Make sure all tests pass successfully

## Affected Areas

`create-package` script

## Risk Analysis

Low risk

## Linked Issues

- https://github.com/little-spoon-dev/design-system/pull/1018

## Checklist

- [x] [Conventional Commits](https://www.conventionalcommits.org/)
- [x] Tests
- [x] Documentation
- [x] Storybook

### CODE REVIEW

**Review by andrii-matviichuk — APPROVED** [review]

File: scripts/create-package.ts [hunk 1]
```diff
@@ -195,9 +195,6 @@ const DEFAULT_COMPONENT_NAME = 'Component'
    * Add package to dependencies.
    */
   if (dependencies) {
-    /*dependencies.forEach((dependency) =>
-      exec(`yarn lerna add ${dependency} --exact --scope=${packageName}`),
-    )*/
     exec(`yarn workspace ${packageName} add ${dependencies.join(' ')} --exact`)
```
Comment (Zaggen) [inline scripts/create-package.ts:198]: I assume this is equivalent to what we had before but in a single `exec`, right?

File: scripts/create-package.ts [hunk 2]
```diff
@@ -295,7 +292,6 @@ function validatePackageName(packageName: string) {
  * @param componentName - Component name.
  */
 function addPackageToUi(packageName: string, directory: string, componentName?: string) {
-  //exec(`yarn lerna add ${packageName} --exact --scope=@littlespoon/ui`)
   const packagePath = resolve(path.packages, directory)
   exec(`yarn workspace @littlespoon/ui add file:${packagePath} --exact`)
```
Comment (Zaggen) [inline scripts/create-package.ts:296]: Can you explain the difference with the commented-out code?

File: scripts/create-package.ts (hunk 1 above)
Comment (huhgawz) [reply scripts/create-package.ts:198]: Yes. That's right

File: scripts/create-package.ts (hunk 2 above)
Comment (huhgawz) [reply scripts/create-package.ts:296]: As you can see in the commented out code, the Lerna `add` command was executed to install/add a dependency to a internal package (`@littlespoon/ui`). Recently, the `add` command was removed from Lerna CLI. Then, in order to achieve what I just described, we now need to use the yarn `add` command

Comment (Zaggen) [reply scripts/create-package.ts:296]: Got it, thanks!

**Review by Zaggen — APPROVED** [review]

**Pull request merged**